src/support/hazard.c

```diff
@@ -28,22 +28,18 @@ hazard_grow(WT_SESSION_IMPL *session)
      * Allocate a new, larger hazard pointer array and copy the contents of the original into place.
      */
     size = session->hazard_size;
-    WT_RET(__wt_calloc_def(session, size * 2, &nhazard));
-    memcpy(nhazard, session->hazard, size * sizeof(WT_HAZARD));
+    WT_RET(__wt_calloc_def(session, size * 2, &new_hazard));
+    memcpy(new_hazard, session->hazard, size * sizeof(WT_HAZARD));
 
     /*
      * Swap the new hazard pointer array into place after initialization is complete (initialization
      * must complete before eviction can see the new hazard pointer array), then schedule the
      * original to be freed.
      */
     ohazard = session->hazard;
```

Comment:
Can we rename to `old_hazard`?

Reply:
Updated in https://github.com/wiredtiger/wiredtiger/pull/9530/commits/5a7969b51aaf7cdd292a92be07e3774d13c7fe6f